src/sentry/static/sentry/app/views/releases/list/index.tsx

```diff
@@ -266,7 +266,9 @@ class ReleasesList extends AsyncView<Props, State> {
       return <EmptyStateWarning small>{t('There are no releases.')}</EmptyStateWarning>;
     }
 
-    return <ReleaseLanding organization={organization} />;
+    return (
+      <ReleaseLanding organization={organization} projectId={selection.projects[0]} />
```

Comment:
This list can be empty (my projects) or (-1 all projects) Is it ok if -1 works its way down to metrics?

Reply:
I filtered it out 👍